tests/oss-fuzz/README.md

```diff
@@ -0,0 +1,80 @@
+# Fuzzing AVIF on oss-fuzz
+
+*Last updated: 2024-01-22* 
+*Authors: [Vincent Rabaud](mailto:vrabaud@google.com)*
+*Status: **Current***
```

Comment:
not sure any of these are needed? we don't usually have them, and all of it can be seen in the git history

Reply:
Done